torjail

```diff
@@ -107,7 +107,7 @@ cleanup() {
     fi
     iptables -D INPUT -i in-$NAME -p udp --destination $IPHOST --dport 5354 -j ACCEPT
     iptables -D INPUT -i in-$NAME -p tcp --destination $IPHOST --dport 9040 -j ACCEPT
-    iptables -D INPUT -i in-$NAME -p udp --destination $IPHOST --dport 9040 -j ACCEPT
+  # iptables -D INPUT -i in-$NAME -p udp --destination $IPHOST --dport 9040 -j ACCEPT
```

Comment:
can you merge all your commits in 1? Thanks!

Reply:
This pull request contains 5 commits, one addressing reliability problem, and the other disables udp support (since Tor doesn't support it). The changes are fully merged in my fork @ https://github.com/harlyh/torjail

Unfortunately here all I can do is wait for Mr. @lesion to act :)